test/pageObjects/createCaseWizardPage.js

```diff
@@ -119,6 +123,16 @@ class CreateCaseWizardPage extends BasePage{
         await this.continueButton.click();
     }
 
+
+    async enterPostcode(postcode) {
+      await element(this.ccdAddressUKField.postcodeText).sendKeys(postcode);
+    }
+
+    async clickFindAddressButton() {
+      await this.ccdAddressUKField.findAdressButton.click();
+      await new CaseDetailsPage()
```

Comment:
don;t think we need this? this is a page object for the Case View page (maybe we should rename this class some other time) so no need to call this class at tthis point as we are still on a case wizard/form page